commands/misc/speedrank.js

```diff
@@ -0,0 +1,62 @@
+const {Command} = require('discord.js-commando');
+const path = require('path');
+speedrankPath = path.join(__dirname,
+  '..', '..', 'utilities', 'speedrank-utils.js');
+const speedrankUtils = require('../../utilities/speedrank-utils.js');
+
+module.exports = class ReplyCommand extends Command {
+  /** constructor for looking up soulbreaks.
+   * @param {Object} client: discord.js-commando client.
+   **/
+  constructor(client) {
+    super(client, {
+      name: 'rank',
+      group: 'misc',
+      memberName: 'speedrank',
+      description: 'Looks up all the entries for a given speedrunner in a specified leaderboard' +
+      ' from the FFRecordKeeper Discord Speedrun challenge leaderboard' +
+      ' (https://docs.google.com/spreadsheets/d/11gTjAkpm4D3uoxnYCN7ZfbiVnKyi7tmm9Vp9HvTkGpw).',
+      examples: ['rank EverythingIsGravy 5star', 'top 10 no-csb', 'top 5 cod', 'top 3 Maliris'],
```

Comment:
Please take out the `'top'` examples from this and add an example for the rank command without a category, ie `rank littlefiredragon`.

Reply:
Done, as requested!